java-backend/pom.xml

```diff
@@ -55,6 +55,11 @@
       <version>2.23.4</version>
       <scope>test</scope>
     </dependency>
+      <dependency>
+          <groupId>org.apache.ant</groupId>
```

Comment:
this is a really big dependency just for splitting a string into words. I'd rather we not pull this in.